var/spack/repos/builtin/packages/silo/package.py

```diff
@@ -64,6 +64,11 @@ def install(self, spec, prefix):
                 'CXXFLAGS={0}'.format(self.compiler.pic_flag),
                 'FCFLAGS={0}'.format(self.compiler.pic_flag)]
 
+        # H5public.h includes mpi.h directly
```

Comment:
This comment about HDF5's public header file is true *only* for installs of HDF5 with parallel support.